src/mscorlib/src/System/Text/Normalization.Windows.cs

```diff
@@ -49,6 +56,13 @@ internal static bool IsNormalized(String strInput, NormalizationForm normForm)
 
         internal static String Normalize(String strInput, NormalizationForm normForm)
         {
+            if (GlobalizationMode.Invariant)
+            {
+                // In Invariant mode we assume all characters are normalized. 
```

Comment:
I have ran the whole tests but didn't enable the flag for all tests. I just enabled the flag for the test I created specifically covering the globalization. so I am not expecting the URL test to fail as we don't enable the invariant mode on such test. this apply to the current globalization tests, without enabling the invariant mode we shouldn't change the behavior. 

I am going to do more tests on Linux as I expect some failures after we have changed the way we initialize ICU.